packages/modules/itwin-viewer-react/package.json

```diff
@@ -55,31 +55,31 @@
     "redux": "^4.0.5"
   },
   "devDependencies": {
-    "@bentley/bentleyjs-core": "^2.6.4",
-    "@bentley/extension-client": "^2.6.4",
-    "@bentley/frontend-application-insights-client": "^2.6.4",
-    "@bentley/frontend-authorization-client": "^2.6.4",
-    "@bentley/geometry-core": "^2.6.4",
-    "@bentley/imodelhub-client": "^2.6.4",
-    "@bentley/imodeljs-common": "^2.6.4",
-    "@bentley/imodeljs-frontend": "^2.6.4",
-    "@bentley/imodeljs-i18n": "^2.6.4",
-    "@bentley/imodeljs-markup": "^2.6.4",
-    "@bentley/imodeljs-quantity": "^2.6.4",
-    "@bentley/itwin-client": "^2.6.4",
-    "@bentley/orbitgt-core": "^2.6.4",
-    "@bentley/presentation-common": "^2.6.4",
-    "@bentley/presentation-components": "^2.6.4",
-    "@bentley/presentation-frontend": "^2.6.4",
-    "@bentley/product-settings-client": "^2.6.4",
-    "@bentley/rbac-client": "^2.6.4",
-    "@bentley/telemetry-client": "^2.6.4",
-    "@bentley/ui-abstract": "^2.6.4",
-    "@bentley/ui-components": "^2.6.4",
-    "@bentley/ui-core": "^2.6.4",
-    "@bentley/ui-framework": "^2.6.4",
-    "@bentley/ui-ninezone": "^2.6.4",
-    "@bentley/webgl-compatibility": "^2.6.4",
+    "@bentley/bentleyjs-core": "2.11.2",
```

Comment:
Probably don't want to pin the versions here.

Reply:
I think the issue here was the breaking changes in 2.12.....this is just devDependencies, though in theory tests should be able to run against all supported versions in peerDeps... @aruniverse we are using the viewer with 2.12.3 in the hub though and it is running without issue. Would it only fail in desktop scenarios?

Reply:
I'm honestly surprised you guys were able to get it working in the hub. I was unable to get the itwin-viewer-react pkg to build locally if i went up to 2.12 cause of the breaking electron changes, which prevented me from running the sample app. I agree tho, i dont like pinning this down here either, but it doesnt hurt. and we'll probably go back to using ^ once we react to the breaking changes in 2.12

Reply:
Okay, I think that should be the highest priority to get itwin-viewer working with 2.13.   Even if that means removing support for Desktop temporarily due to the breaking changes since it's broken at this point...

However, the minimum will need to updated to 2.13 once the desktop workflow is supported...